libs/ui/breadcrumb/helm/src/lib/breadcrumb-separator.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, computed, input } from '@angular/core';
+import { HlmIconComponent, provideIcons } from '@spartan-ng/ui-icon-helm';
+import { lucideChevronRight } from '@ng-icons/lucide';
+import { hlm } from '@spartan-ng/ui-core';
+
+@Component({
+   
+  selector: 'hlm-breadcrumb-separator',
+  standalone: true,
+  imports: [HlmIconComponent],
+  providers: [provideIcons({ lucideChevronRight })],
+  host: {
+    '[class]': '_computedClass()',
+  },
+  template: `
+    <div #ref>
+      <ng-content />
+    </div>
+    @if (!ref.hasChildNodes()) {
+      <hlm-icon name="lucideChevronRight" class="h-3.5 w-3.5" />
+    }
```

Comment:
I forgot about this feature in v18. You can specify a fallback content for `ng-content`.

```suggestion
    <ng-content>
	<hlm-icon name="lucideChevronRight" class="h-3.5 w-3.5" />
     </ng-content>
```

Reply:
Oh I missed a great feature of v18. 
Thanks a lot 😄

Reply:
No problem, i totally forgot about it too.